root/edit/search_macros.tt

```diff
@@ -253,6 +253,35 @@
   </select>
 [% END %]
 
+[% MACRO predicate_vote_count(field, field_contents) WRAPPER wfield predicate="vote_count" %]
+  <select name="args.0" class="arg">
+    [% FOR item=[ [  2,   lp('Approve', 'vote') ]
+                , [  1,   lp('Yes', 'vote') ]
+                , [  0,   lp('No', 'vote') ]
+                , [ -1,   lp('Abstain', 'vote') ]
+                ] %]
+    <option value="[% html_escape(item.0) %]"
+            [% 'selected="selected"' IF field_contents.find_argument(item.0).defined %]
+            >[% html_escape(item.1) %]</option>
+    [% END %]
+  </select>
+
+  [% operators([ [ '=', l('is') ],
```

Comment:
Moving vote type to a named parameter allowed to have the exact same HTML code and to deduplicate it in https://github.com/metabrainz/musicbrainz-server/pull/497/commits/3afe77594eb5359c98feb72bc8e329f66cf3a095.